Gio-camacho/week-5/resume-fullstack/frontend/src/app/component/cv/cv.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-cv',
+  templateUrl: './cv.component.html',
+  styleUrls: ['./cv.component.scss']
+})
+export class CvComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```

Comment:
This component seems like is empty and useless. Let me know if I'm wrong

Reply:
true my friend. Many thanks.